core/modules/update/update.fetch.inc

```diff
@@ -332,6 +332,13 @@ function _update_build_fetch_url($project, $site_key = '') {
     $url .= '&list=';
     $url .= rawurlencode(implode(',', $list));
   }
+
+  // Fall back to the less secure HTTP version of the URL (if the respective
+  // setting in settings.php has been enabled).
+  if (settings_get('update_fetch_with_http_fallback', FALSE)) {
```

Comment:
Hm... that's actually not a "fallback", but the insecure connection is *always* used if the setting is on. Confusing variable name and comment wording.

Reply:
I'm using the same wording and setting name as those used in D8/9. @indigoxela are you suggesting that the setting name should be `'update_fetch_with_http'` instead?

Reply:
Renamed to `'update_fetch_with_http'` and tweaked the comment accordingly.